src/features/security/SecurityScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, memo, useRef, useMemo, useState } from 'react'
+import { Animated, ImageStyle, Image } from 'react-native'
+
+const AnimatedImage = Animated.createAnimatedComponent(Image)
+
+type Props = { visible: boolean }
+const SecurityScreen = ({ visible }: Props) => {
+  const fadeAnim = useRef(new Animated.Value(0))
+  const [mounted, setMounted] = useState(false)
+
+  const fadeIn = () => {
+    setMounted(true)
+    Animated.timing(fadeAnim.current, {
+      toValue: 1,
+      duration: 500,
```

Comment:
I think this needs to be a little faster. Sometimes when I minimize on iOS I can still see the content and the security screen is slightly faded above it. When I set it to `100` it seems to work

Reply:
I think whats happening is maybe the animation stops as it goes to background?

Reply:
Hmm it also might be the map slowing down the animation thread? I think it happens on when I am also still loading the map. Might be a debug only issue

Reply:
Hmm also doesnt seem to be working on Android. But the visible boolean is getting set correctly. My android device is pretty slow though.. but I'm not seeing it ever appear

Reply:
You're right about Android. It appears to not be working in the old app either. I'll investigate.

Reply:
Fixed with your new changes